docs/src/modules/java/nav.adoc

```diff
@@ -1,11 +1,10 @@
 ** xref:java:index.adoc[]
-*** xref:java:developer-tools.adoc[]
-*** xref:developing:development-process-java.adoc[]
-*** xref:java:quickstart-template.adoc[]
-*** xref:java:proto.adoc[]
+*** xref:java:getting-started.adoc[Getting Started]
+*** xref:java:spring-boot-integration.adoc[Spring Boot Integration]
+*** xref:java:development-process-spring.adoc[Development Process]
```

Comment:
maybe change the name of this file to `development-process.adoc`?

Reply:
changed, tks